.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -4,12 +4,10 @@ about: 'Suggest a new idea or feature'
 labels: 'enhancement'
 ---
 
-Type here...
+Thanks for taking the time to suggest a new feature!
 
----
-
-Please follow these steps to make it more efficient to respond to your feature request.
+Please read the guidelines below to make it more efficient to respond to your feature request, and then delete this message before submitting the issue.
```

Comment:
See my comment on the pull request template on using the comments syntax to hide the message by default when submitting the issue.